packages/boba/contracts/deployment/rinkeby/addresses.json

```diff
@@ -0,0 +1,59 @@
+{
+  "TOKENS": {
+    "TEST": {
+      "L1": "",
+      "L2": ""
+    },
+    "USDT": {
```

Comment:
add Boba governance token to list for testing?

Reply:
@souradeep-das is on it!